README.md

```diff
@@ -55,9 +52,23 @@ Cockpit Composer is built using [React](https://reactjs.org/). For inspecting th
 
     $ npm run build:debug
 
-### Running end to end test
+### Running integration test
 
-[End-to-End Test Running Guide](test/end-to-end/README.md).
+Run test without visually seeing what the browser is doing:
+
+    $ make check
+
+In the event you wish to visually see what the browser is doing you will want to run:
+
+    $ TEST_SHOW_BROWSER=true make check
+
+By default the cockpit-composer will be installed into Fedora 31 and test will be run on Chromium. To run it on Firefox, a prefix ```TEST_BROWSER=firefox``` needs to be added, like:
+
+    $ TEST_BROWSER=firefox make check
+
+To test cockpit-composer in different OS, set the `$TEST_OS` environment variable, for example:
```

Comment:
A few comments on the readme:
1. `In the event you wish to visually` -> `In the case you wish to visually`
2. There is also `make debug-check` which does the same. Worth mentioning it here or dropping it?
3. `will be installed into Fedora 31` - that is true now, but not sure if you want to keep this up to date all the time. Maybe better to point to `TEST_OS` in Makefile.
4. `To run it on Firefox, a prefix` - it is not a prefix, it is an `environment variable`.